integrations/github/client.go

```diff
@@ -246,5 +246,18 @@ func New(sec sdkservices.Secrets) sdkservices.Integration {
 			sdkmodule.WithFuncDoc("https://docs.github.com/en/rest/users#get-a-user"),
 			sdkmodule.WithArgs("username"),
 		),
+		// Actions
+		sdkmodule.ExportFunction(
+			"list_workflows",
+			i.listWorkflows,
+			sdkmodule.WithFuncDoc("https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event"),
+			sdkmodule.WithArgs("owner", "repo"),
```

Comment:
Can you also add "per_page" and "page"? (Both optional integers). I expect realistic usage of this will easily go over the default (1 page of 30 results).
Anyway, not as urgent as the core capability, so feel free to do it in a separate PR.

Reply:
done